test/active_record_test.rb

```diff
@@ -179,28 +177,21 @@ def test_should_set_attributes_regardless_of_arguments_order
     Account.new.attributes = { password: "password", key: SECRET_KEY }
   end
 
-  def test_should_create_changed_predicate
-    person = Person.create!(email: 'test@example.com')
-    refute person.email_changed?
-    person.email = 'test@example.com'
-    refute person.email_changed?
-    person.email = nil
-    assert person.email_changed?
-    person.email = 'test2@example.com'
-    assert person.email_changed?
-  end
-
   def test_should_create_was_predicate
     original_email = 'test@example.com'
     person = Person.create!(email: original_email)
-    assert_equal original_email, person.email_was
+    assert_equal nil, person.email_was
```

Comment:
I was surprised to see the old assertion pass as email previously had no value. This new assertion seems correct to me.